### ISSUE TITLE
Installation issues with Python 3.13

### ISSUE THREAD

**Issue by smaret** [radio-astro-tools/statcont#10]
Installation with Python 3.13.0 fail with the following error:

```terminal
python3 setup.py install
```
```python
Traceback (most recent call last):
  File "/Users/smaret/Documents/Software/statcont/setup.py", line 8, in <module>
    import ah_bootstrap
  File "/Users/smaret/Documents/Software/statcont/ah_bootstrap.py", line 36, in <module>
    import imp
ModuleNotFoundError: No module named 'imp'
```

**Comment by smaret** [comment]
I see there is a PR that should solve this (#9 )

**Comment by smaret** [comment]
Fixed with #9
